index.bs

```diff
@@ -3434,24 +3433,22 @@ interfaces that have interface members with these names.
 
     <blockquote>
 
-        The <a class="dfnref" href="#">values to iterate over</a>
-        are the open <code class="idl">Session</code> objects
-        on the <code class="idl">SessionManager</code> sorted by username.
+        The [=value pairs to iterate over=] are the list of key-value pairs with the key being the username and the
+        value being the open <code class="idl">Session</code> object on the <code class="idl">SessionManager</code>
+        object corresponding to that username, sorted by username.
 
-        <p class="issue">
-            Fix reference to removed definition for "values to iterate over".
-        </p>
     </blockquote>
 
     In the ECMAScript language binding, the [=interface prototype object=]
     for the <code class="idl">SessionManager</code> [=interface=]
     has a <code>values</code> method that is a function, which, when invoked,
     returns an iterator object that itself has a <code>next</code> method that returns the
-    next value to be iterated over.  It has <code>values</code> and <code>entries</code>
-    methods that iterate over the indexes of the list of session objects
-    and [index, session object] pairs, respectively.  It also has
+    next value to be iterated over.  It has <code>keys</code> and <code>entries</code>
+    methods that iterate over the usernames of session objects
+    and [username, Session object] pairs, respectively.  It also has
```

Comment:
Can you use the tuple/pair notation from infra here (either `(username, Session object)` or `username/Session object`)?